src/server/pfs/server/driver.go

```diff
@@ -2132,7 +2140,7 @@ func (d *driver) deleteAll(ctx context.Context) error {
 			}
 		}
 		for _, projectInfo := range projectInfos {
-			if err := d.deleteProject(txnCtx, projectInfo.Project, true); err != nil {
+			if err := d.deleteProject(ctx, &pfs.DeleteProjectRequest{Project: projectInfo.Project, Force: true}); err != nil {
```

Comment:
Hmmm, the change to create the transaction with `deleteProject` means that deleting N projects will create N+1 transactions. I think it would be good to continue to pass in the transaction. It makes the signature unwieldy (`context.Context, txncontext.TransactionContext, …`), but that is life.